builtin/credential/github/cmd/github/main.go

```diff
@@ -0,0 +1,29 @@
+package main
```

Comment:
This one doesn't build. I'm getting:

```
../../path_login.go:217:43: client.Organizations.ListUserTeams undefined (type *github.OrganizationsService has no field or method ListUserTeams)
```

The rest build fine though.

Reply:
I just merged in master, did `go build`, and confirmed that it's fixed.